bindings/python/examples/exchange/1_create_account.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+# This example creates a new database and account.
+
+from iota_sdk import Wallet, StrongholdSecretManager, SyncOptions, CoinType
+from dotenv import load_dotenv
+import os
+
+# This example uses secrets in environment variables for simplicity which should not be done in production.
+load_dotenv()
+
+if 'WALLET_DB_PATH' not in os.environ:
+    raise Exception(".env WALLET_DB_PATH is undefined, see .env.example")
+if 'NODE_URL' not in os.environ:
+    raise Exception(".env NODE_URL is undefined, see .env.example")
+if 'STRONGHOLD_SNAPSHOT_PATH' not in os.environ:
+    raise Exception(
+        ".env STRONGHOLD_SNAPSHOT_PATH is undefined, see .env.example")
+if 'STRONGHOLD_PASSWORD' not in os.environ:
+    raise Exception(".env STRONGHOLD_PASSWORD is undefined, see .env.example")
+if 'MNEMONIC' not in os.environ:
+    raise Exception(".env MNEMONIC is undefined, see .env.example")
+
+client_options = {
+    'nodes': [os.environ.get('NODE_URL')],
+}
+
+secret_manager = StrongholdSecretManager(
+    os.environ.get('STRONGHOLD_SNAPSHOT_PATH'), os.environ['STRONGHOLD_PASSWORD'])
+
+wallet = Wallet(os.environ.get('WALLET_DB_PATH'),
+                client_options, CoinType.IOTA, secret_manager)
+
+# Store the mnemonic in the Stronghold snapshot, this only needs to be done once
+wallet.store_mnemonic(
+    os.environ['MNEMONIC'])
+
+created_account = wallet.create_account('Alice')
+account = wallet.get_account('Alice')
```

Comment:
Can't we just use created_account ?

Reply:
Currently not, but we should change that

Reply:
Can you open an issue please?

Reply:
Opened an issue so we don't forget it https://github.com/iotaledger/iota-sdk/issues/743

Reply:
Thanks